src/main/java/aztech/modern_industrialization/machines/multiblocks/MultiblockMachineBlockEntity.java

```diff
@@ -59,7 +115,7 @@ public boolean useWrench(Player player, InteractionHand hand, BlockHitResult hit
     }
 
     @Override
-    public final void setRemoved() {
+    public void setRemoved() {
```

Comment:
Why not keep this final? When you make this kind of change it really needs to be explained. 😅

Reply:
Also used for my tesla tower :P I need to unload the network when the machine is destroyed in world

Reply:
Ok that's reasonable. :P